fendermint/vm/interpreter/src/fvm/genesis.rs

```diff
@@ -189,17 +189,6 @@ where
             )
             .context("failed to create cron actor")?;
 
-        // Ethereum Account Manager (EAM) actor
-        state
-            .create_builtin_actor(
-                eam::EAM_ACTOR_CODE_ID,
-                eam::EAM_ACTOR_ID,
-                &EMPTY_ARR,
-                TokenAmount::zero(),
-                None,
-            )
-            .context("failed to create EAM actor")?;
-
         // Burnt funds actor (it's just an account).
```

Comment:
Is there any harm in keeping this here? The `replace` method should be able to override this.

Reply:
Yes, please keep it. If we open up replacements to users, we don't want to modify this part of the code, so we should walk that path here as well IMO.

Reply:
correct, but it should work without this as well.

Reply:
True, it works without it - the question is, does it work _with_ it?